app/pages/admin/featured-project-toggle.jsx

```diff
@@ -2,23 +2,31 @@ import PropTypes from 'prop-types';
 import React from 'react';
 
 function FeaturedProjectToggle({ error, handleProjectChange, project }) {
+  if (project.launch_approved) {
+    return (
+      <div>
+        <h4>Featured project</h4>
+        <label className="">
+          <input
+            disabled={project.featured}
+            checked={project.featured}
+            name="featured"
+            onChange={handleProjectChange.bind(this)}
```

Comment:
What is `.bind(this)` doing here?

Reply:
I thought that since the function deals with events needed to be bound. But it looks like it isn't needed after all.

Reply:
It would bind the function to `FeaturedProjectToggle` but that’s not where the function is invoked, so it confused me. I think it should be bound before it is passed down as a prop.

Reply:
It is [bound in the parent component](https://github.com/zooniverse/Panoptes-Front-End/blob/8ed377c0288e2e62ea65251121de0b7980b95a3e/app/pages/admin/project-status.jsx#L29). That's why it's not needed in `FeaturedProjectToggle`, thanks for spotting it.